src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -185,7 +192,16 @@
           Background="{DynamicResource {x:Static vsShell:VsBrushes.BrandedUIBackgroundKey}}"
           vsTheming:ImageThemingUtilities.ImageBackgroundColor="{Binding Background, RelativeSource={RelativeSource Self}, Converter={StaticResource BrushToColorConverter}}">
         <Grid x:Name="Overlay" Style="{StaticResource OverlayGridStyle}" Panel.ZIndex="1000">
-            <TextBlock Style="{StaticResource OverlayTextStyle}" Text="Please select an issue from the Error List" />
+            <StackPanel VerticalAlignment="Center">
+                <TextBlock Style="{StaticResource OverlayTitleTextStyle}" Text="Please select a SonarLint issue with secondary locations from the Error List."/>
+                <TextBlock Style="{StaticResource OverlayTextStyle}">
+                    <Hyperlink NavigateUri="https://github.com/SonarSource/sonarlint-visualstudio/wiki" 
+                               RequestNavigate="Hyperlink_OnRequestNavigate">
+                        <LineBreak/>
+                        <Run Text="Click here to go online and find out more about this and other SonarLint features." />
```

Comment:
@duncanp-sonar please review the message 😆

Reply:
Looks ok to me.